core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.html

```diff
@@ -0,0 +1,117 @@
+<form [formGroup]="form">
+    <p-accordion [activeIndex]="0" expandIcon="pi pi-angle-down" collapseIcon="pi pi-angle-up">
+        <p-accordionTab
+            [disabled]="isLoading"
+            [header]="'block-editor.extension.ai-image.settings' | dm"
+            iconPos="end">
+            <div class="field ai-image-prompt__input">
+                <label dotFieldRequired>{{ 'block-editor.extension.ai-image.input' | dm }}</label>
+                <div>
+                    <p-radioButton
+                        formControlName="type"
+                        value="input"
+                        inputId="optionOne"></p-radioButton>
+                    <label class="ml-2" for="optionOne">{{
+                        'block-editor.extension.ai-image.custom.prompt' | dm
+                    }}</label>
+                </div>
+                <span>{{ 'block-editor.extension.ai-image.custom.prompt.desc' | dm }}</span>
+                <div>
+                    <p-radioButton
+                        formControlName="type"
+                        value="auto"
+                        inputId="option2"></p-radioButton>
+                    <label class="ml-2" for="option2">{{
+                        'block-editor.extension.ai-image.existing.content' | dm
+                    }}</label>
+                </div>
+                <span>{{ 'block-editor.extension.ai-image.existing.content.desc' | dm }}</span>
+            </div>
+            <div class="field ai-image-prompt__orientation">
+                <label dotFieldRequired>{{
+                    'block-editor.extension.ai-image.orientation' | dm
+                }}</label>
+                <p-dropdown
+                    [options]="orientationOptions"
+                    formControlName="size"
+                    styleClass="w-full"></p-dropdown>
+            </div>
+        </p-accordionTab>
+        <p-accordionTab
+            [disabled]="isLoading"
+            [header]="'block-editor.extension.ai-image.prompt' | dm"
+            iconPos="end">
+            <div
+                class="ai-image-prompt__text"
+                [class.ai-image-prompt__text--generated]="aiProcessedPrompt && !isLoading">
+                <div class="field">
+                    <label [checkIsRequiredControl]="'size'" dotFieldRequired>{{
+                        (form.get('type').value === 'input'
+                            ? 'block-editor.extension.ai-image.prompt'
+                            : 'block-editor.extension.ai-image.custom.props'
```

Comment:
can we make an static property for this? I think this `form.get` is triggering in every render.

Reply:
done